tests/data/qualifier_cases.json

```diff
@@ -501,7 +501,7 @@
             "Plausibility.Hypothetical",
             "Negation.Affirmed",
             "Temporality.Current",
-            "Experiencer.Patient"
+            "Experiencer.Other"
```

Comment:
Nice catch

Reply:
Will add a line to the change log. Yeah, for sure I need to refactor this because of the duplication. I will make it a separate git issue :).